### PR TITLE
[#1624] Fixed rename refactoring with inferred JVM elements

### PR DESCRIPTION
**Pull request by szarnekow**

### CODE REVIEW

**Comment by szarnekow** [comment]
Alternative to the attempted fix from https://github.com/eclipse/xtext-eclipse/pull/1624

File: org.eclipse.xtext.ui/src/org/eclipse/xtext/ui/refactoring/impl/DefaultDependentElementsCalculator.java [hunk 1]
```diff
@@ -41,23 +41,31 @@
...
 			}
 		}
 		return elementURIs;
 	}
+	
+	/**
+	 * @since 2.25
+	 */
+	protected boolean tryAddDependentElement(EObject element, List<URI> result) {
+		if (nameProvider.getFullyQualifiedName(element)!=null) {
```
Comment (cdietrich) [inline org.eclipse.xtext.ui/src/org/eclipse/xtext/ui/refactoring/impl/DefaultDependentElementsCalculator.java:61]: inconsistent formatting

**Comment by cdietrich** [comment]
@szarnekow should we move this to 2.26?

**Comment by cdietrich** [comment]
this repository has been merged/moved to https://github.com/eclipse/xtext.
please feel free to recreate this PR against that repository in case it still is relevant.

**Pull request closed without merging**